lib/autoproj/cli/version.rb

```diff
@@ -0,0 +1,62 @@
+require 'autoproj/cli/base'
+require 'rubygems'
+
+module Autoproj
+module CLI
+    class Version < Base
+        # List the version of autoproj and optionally include
+        # the installed dependencies with information about the requirement
+        # and the actual used version
+        def run(args, options = Hash.new)
+            puts "autoproj version: #{Autoproj::VERSION}"
+            if options[:deps]
```

Comment:
Please return early, that is

~~~
return unless options[:dep]
~~~

Reply:
Ping comment above

Reply:
sorry I missed that one